processes/development/example-mapping-and-estimating-process.md

```diff
@@ -0,0 +1,215 @@
+
+##### Required reading and viewing
+
+- [Original example mapping article (PDF)](https://drive.google.com/a/okgrow.com/file/d/12I8KwqgWZXUY6C9zXyTKxGpH3xDp6EtE/view?usp=sharing)
+- [Agile Estimating - Mike Cohn (Video)](https://www.youtube.com/watch?v=37zfyncCpkA)
+
+##### Strongly recommended
+
+- Short book (80 pages):  [*Discovery: Explore Behaviour Using Examples*](https://drive.google.com/a/okgrow.com/file/d/1MRTXD-I8GME4Cp6RzRAubMt8U-emfL63/view?usp=sharing)
+
+- Chapter 6 from Agile Estimating and Planning (12 pages): [Estimating Techniques](https://drive.google.com/a/okgrow.com/file/d/10IsJMmfMythKIUj_wBG-rZz0KT44IHwF/view?usp=sharing)
+
+  ​
+
+## Inputs / Starting resources
+
+- ##### Storymap
+
+  - Use [StoriesOnBoard](https://okg-grow.storiesonboard.com)
+  - [Sample story map (MiOrder)](https://ok-grow.storiesonboard.com/m/storymap-sample-miorder) 
+
+- ##### Trello Board sync'ed to StoriesOnBoard storymap
+
+  - StoriesOnBoard doc: [Setting Up Sync with Trello](http://blog.storiesonboard.com/setting_up_synchronization_with_trello/)
+
+
+- ##### Google Drive Example Map File
+
+  - [Sample Example Map file (MiOrder)](https://docs.google.com/a/okgrow.com/spreadsheets/d/1Wf4jsJFz3IiIV2KpTFoYGVMShe9lf0V2DhjfObVI9ew/edit?usp=sharing)
+
+
+- ##### Members of Project Team
+
+  - At least 2 from Dev team
+  - Client will be expected to participate, but may not be practical to ask for 5+ hours per day for 1 week
+
+- ##### Scheduled Daily (if possible) Client Discovery Meetings (30 minutes)
+
+- ##### Scheduled Example Mapping Workshops
+
+  - How long?
+
+    Full day
+
+    Break up the day into 2 sections: 3hrs each broken by lunch break
+
+  - How many?
+
+    Depends on length of Discovery Phase.
+
+  - Account for time differences and time zones
+
+  - Break regularly - aim for timeboxed 25-minute example mapping sessions. Take a 5-10 minute Break every 2 sessions or if a session goes longer than 45 minutes
+
+  ##### 
+
+## Process
+
+- #### Internal Kickoff
+
+  ##### As a group
+
+  - ##### Walkthrough Storymap
+
+    Gain an understanding of the Project goals, value props.  Good input (if available) is any Design Sprint output.
+
+  - ##### Red-flag Critical / Risky / Ambiguous `Task`-level cards
+
+    - Any cards that are too vague / ambiguous
+    - Any cards involving unknown API or partners
+    - Any cards that involve a technology that we don't have experience in
+
+  - ##### For each flagged card
+
+    1. Come to a relative order from highest to lowest risk
+    2. Determine if the client is required for the Example Mapping session for the card
+
+- #### Example Mapping
+
+  ##### For each `Task`-level card in Storymap (TIMEBOX 25 minutes)
+
+  1. Identify the card, provide context
+
+     - As much as possible, address the riskiest cards as soon as possible
+     - Deviate from the risk-priority only if doing a lower-priority card is needed to provide context
```

Comment:
This section sounds linear, but I think the experience is not. I would suggest mentioning something like:

`Each card will be different. It may be easiest to start with obvious rules, but in some cases you can use examples to discover a rule that's hard to formulate`

i.e. - it's fine to start with either rules or examples, and going back and forth will help clarify things.

Reply:
Also, I'd maybe stick with bullets instead of numbers to reinforce that this isn't really a perfect do-this-before-that process.

Reply:
Good notes.  I'll make those changes